client/src/hooks/appSyncHooks.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react'
+import { API, GraphQLQuery } from '@aws-amplify/api'
+//import { userBySub } from '../graphql/queries'
+import * as queries from '../graphql/generatedQueries'
+import { UserBySubQuery } from '../types/generatedAPI';
+
+export const useGetUserData = (userSub: string) => {
```

Comment:
We can keep this hook and why not use `useAuthUser()` here, instead of passing it as a parameter?

Reply:
Yeah, we can use it there

Reply:
Great 👍